pyteal/ir/ops.py

```diff
@@ -27,120 +30,121 @@ def min_version(self) -> int:
-    min_balance       = OpType("min_balance",       Mode.Application,                  3)
-    pushbytes         = OpType("pushbytes",         Mode.Signature | Mode.Application, 3)
-    pushint           = OpType("pushint",           Mode.Signature | Mode.Application, 3)
-    shl               = OpType("shl",               Mode.Signature | Mode.Application, 4)
-    shr               = OpType("shr",               Mode.Signature | Mode.Application, 4)
-    sqrt              = OpType("sqrt",              Mode.Signature | Mode.Application, 4)
-    bitlen            = OpType("bitlen",            Mode.Signature | Mode.Application, 4)
-    exp               = OpType("exp",               Mode.Signature | Mode.Application, 4)
-    divmodw           = OpType("divmodw",           Mode.Signature | Mode.Application, 4)
-    expw              = OpType("expw",              Mode.Signature | Mode.Application, 4)
-    b_add             = OpType("b+",                Mode.Signature | Mode.Application, 4)
-    b_minus           = OpType("b-",                Mode.Signature | Mode.Application, 4)
-    b_div             = OpType("b/",                Mode.Signature | Mode.Application, 4)
-    b_mul             = OpType("b*",                Mode.Signature | Mode.Application, 4)
-    b_lt              = OpType("b<",                Mode.Signature | Mode.Application, 4)
-    b_gt              = OpType("b>",                Mode.Signature | Mode.Application, 4)
-    b_le              = OpType("b<=",               Mode.Signature | Mode.Application, 4)
-    b_ge              = OpType("b>=",               Mode.Signature | Mode.Application, 4)
-    b_eq              = OpType("b==",               Mode.Signature | Mode.Application, 4)
-    b_neq             = OpType("b!=",               Mode.Signature | Mode.Application, 4)
-    b_mod             = OpType("b%",                Mode.Signature | Mode.Application, 4)
-    b_or              = OpType("b|",                Mode.Signature | Mode.Application, 4)
-    b_and             = OpType("b&",                Mode.Signature | Mode.Application, 4)
-    b_xor             = OpType("b^",                Mode.Signature | Mode.Application, 4)
-    b_not             = OpType("b~",                Mode.Signature | Mode.Application, 4)
-    bzero             = OpType("bzero",             Mode.Signature | Mode.Application, 4)
-    gload             = OpType("gload",             Mode.Application,                  4)
-    gloads            = OpType("gloads",            Mode.Application,                  4)
-    gaid              = OpType("gaid",              Mode.Application,                  4)
-    gaids             = OpType("gaids",             Mode.Application,                  4)
-    callsub           = OpType("callsub",           Mode.Signature | Mode.Application, 4)
-    retsub            = OpType("retsub",            Mode.Signature | Mode.Application, 4)
+    err = OpType("err", Mode.Signature | Mode.Application, 2)
```

Comment:
This makes me sad but I suppose there's no way to preserve the special spacing here

Reply:
What about this: https://stackoverflow.com/questions/58584413/black-formatter-ignore-specific-multi-line-code ?

Reply:
The special formatting can stay. Thanks to @ahangsu.